bundles/persistence/org.eclipse.smarthome.persistence.mapdb/src/main/java/org/eclipse/smarthome/persistence/mapdb/internal/MapDbItem.java

```diff
@@ -0,0 +1,80 @@
+/**
+ * Copyright (c) 2010-2016 by the respective copyright holders.
```

Comment:
Please update all license herders to EPL 2. Run the command 

```
mvn license:format -pl bundles/persistence/org.eclipse.smarthome.persistence.mapdb
```

from the project root directory to have this done automatically.